alternatives.md

```diff
@@ -0,0 +1,129 @@
+# Babelfish vs Other Software
+
+Table of Contents
+=================
+
+* [Kythe](#kythe)
+* [Language Server Protocol](#language-server-protocol)
+* [srclib](#srclib)
+* [ctags](#ctags)
+* [ANTLR](#antlr)
+* [Tree\-sitter](#tree-sitter)
+* [srcML](#srcml)
+* [SmaCC](#smacc)
+
+<!-- TODO: https://github.com/oracle/opengrok/wiki/Comparison-with-Similar-Tools -->
+
+## [Kythe](http://kythe.io/)
+
+> The best way to view Kythe is as a “hub” for connecting tools for various languages, clients and build systems. By defining language-agnostic protocols and data formats for representing, accessing and querying source code information as data, Kythe allows language analysis and indexing to be run as services.
+
+> Some tools (e.g., static analyzers) already have expressive purpose-built internal representations for code. Kythe is not meant to be a universal replacement for such IRs — instead, our goal is to provide a way for such tools to capture “interesting subsets” of an analysis for sharing with other tools.
+
+Babelfish and Kythe share a goal of defining a common representation
+for concepts from different programming languages and provide a way to query it.
+Both provide a unified data format. And both are language-independent.
+
+The main difference is that Babelfish preserves all AST nodes including
+control flow and expressions, while Kythe focuses on class hierarchy,
+dependencies, etc.
```

Comment:
Rather than class hierarchy I'd say it focuses on object resolution, linking usages of identifiers to their original definition.